test/test_utils/__init__.py

```diff
@@ -1393,6 +1393,10 @@ def get_framework_from_image_uri(image_uri):
         else None
     )
 
+def is_trcomp_image(image_uri):
+    framework = get_framework_from_image_uri(image_uri)
+    return framework == "huggingface_tensorflow_trcomp" or framework == "huggingface_pytorch_trcomp"
```

Comment:
Why check if it is exactly one of these, rather than just checking for `tcomp` as a substring in the framework name?

Reply:
updated